Assignment/Noura Aldablan/NouraD.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+
+<meta charest="utf-8">
+  <title>Noura's Page</title>
+  <link rel="stylesheet" href="css/NouraD.css">
+
+  </head>
+
+  <body>
+
+    <header>
+
+      <ul>
+        <li><a href="file:///C:/Users/DELL/Desktop/Noura%20Aldablan/NouraD.html"><i>About me</i></a> </li>
+        <li class="line"> <a href="file:///C:/Users/DELL/Desktop/Noura%20Aldablan/NouraS.html"><i>Profile</i></a></li>
+      </ul>
+
+      <h1>Noura Aldablan</h1>
+    </header>
+<div class="first">
+
+  <h3>  I Enjoy The life as A Developer</h3>
+<p class="low" > <img src="http://www.lexsa.ir/wp-content/uploads/2018/12/default_avatar.jpg" alt="pic" style="width:170px;height:170px;margin-left:15px;">
+I'm Noura Aldablan, a beginner Developer based in Riyadh. I'm still learning FEW Development with GA-Misk at Alfaisal University
+ because I'm trying to be specialized in this field.
+ </p>
+
+</div>
+
+<div class="good">
+
+</div>
+
+
+  <div class="Second">
+
+  <p> <br/> <br/> <br/> <br/> <br/> <br/> <br/>
```

Comment:
The good way to make the space and newline by <br> , but I would like to say better to use 'margin' to make space between elements.